hw4/hw-lesson-4.py

```diff
@@ -0,0 +1,15 @@
+"""
+Homework-4: text type, even/odd, length
+"""
+
+text = input('Please enter a word or a number - ')
+a = text
+if text.isdigit():
+    text = int(text)
+    x = text % 2
```

Comment:
There is no big need to create a separate variable here. You can put `text % 2` right into `if` condition